Classes/Fluid/ViewHelper/ComponentRenderer.php

```diff
@@ -391,10 +391,11 @@ protected function initializeComponentParams()
                     ), 1532960145);
                 }
 
+                $componentArgumentConverter = $this->getComponentArgumentConverter();
```

Comment:
This creates an instance of `ComponentArgumentConverter` within the loop (as before). You could create the instance once above the loop.